cli/tre/commands/login.py

```diff
@@ -95,7 +97,7 @@ def login_device_code(base_url: str, client_id: str, aad_tenant_id: str, api_sco
 
     app = msal.PublicClientApplication(
         client_id=client_id,
-        authority=f"https://login.microsoftonline.com/{aad_tenant_id}",
+        authority=AuthorityBuilder(AZURE_PUBLIC, aad_tenant_id),
         token_cache=cache)
```

Comment:
This looks similar to the code block above. Have you thought about making a method out of this?
When we introduce support for the gov cloud, it will make even more sense...

Reply:
moved to shared method